### PR TITLE
refactor: separate case events json files in definition (cmc-1038)

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[CMC-1038](https://tools.hmcts.net/jira/browse/CMC-1038)

### Change description ###
This PR splits one CaseEvent json file into related separate Json files under CaseEvent directory.

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "DISPATCH_BUSINESS_PROCESS",
+    "Name": "Dispatch business process",
+    "Description": "Set READY business process status to DISPATCHED",
+    "DisplayOrder": 22,
```
Comment (pd-dudkiewicz) [inline ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json]: Should we remove `DisplayOrder` for any event that's not user event?

Comment (swalker125) [reply ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json]: Agree on the above, a big benefit of this is that we can accurately keep track of where events will be displayed for the user. A further improvement could be to split user events by state, but not a part of this :)

**Review by dharmendrak — DISMISSED** [review]

File: ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json (hunk 1 above)
Comment (sabahirfan) [reply ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json]: Sounds fine as it is irrelevant. It's not part of this tech debt. but I can take care of that as well.

Comment (swalker125) [reply ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json]: @sabahirfan I would ignore state for now, as there is a big state refactor coming :)

**Review by pd-dudkiewicz — APPROVED** [review]

**Review by douglasrice — APPROVED** [review]

**Pull request merged**